src/test/java/LogOutTest.java

```diff
@@ -9,6 +9,7 @@ public class LogOutTest extends BaseTest {
 
     @Test
     void logoutTest() {
+        welcomePage.makeHomePageScreenshot();
```

Comment:
it would be better to use this method after successful login to get a screenshot of the result